opent2t-reference/main.js

```diff
@@ -0,0 +1,283 @@
+var electron = require('electron');
+var app = electron.app;
+var BrowserWindow = electron.BrowserWindow;
+var q = require('q');
+var fs = require('fs');
+require('electron-reload')(__dirname);
+var Opent2tHelper = require("./Opent2tHelper");
+var opent2tHelper = new Opent2tHelper();
```

Comment:
k thanks

---
In reply to: [101380727](https://github.com/openT2T/opent2t-cli/pull/16#discussion_r101380727) [](ancestors = 101380727,101374738)